go.mod

```
@@ -1,11 +1,10 @@
-// change the module path to your directory
-module github.com/polipopoliko/tsaving/tsaving
+module github.com/david1312/tsaving
```

```suggestion
module github.com/ndv6/tsaving
```
change to match github repository link